experiment/references.md

```diff
@@ -5,13 +5,30 @@
 3. Colin N. Banwell, Elaine M. McCash (1994). Fundamentals of Molecular Spectroscopy. Mcgraw hill.
 
 **Other references:**
+<!-- No working -->
+<!-- 
+1. [http://orgchem.colorado.edu/hndbksupport/specttutor/irchart.html](http://orgchem.colorado.edu/hndbksupport/specttutor/irchart.html) -->
+
+<!-- domain is not correct -->
+<!-- 1. [ Phase Diagrams](https://chem-ilp.net/labTechniques/IRVideo.htm) -->
+
+1. [Spectral Mode Identification](https://www.umass.edu/microbio/chime/ir-spect/#)
+
+<!-- Page not found -->
+<!-- 3. [http://www.chem.umass.edu/~nermmw/Spectra/irspectra/Methanol/methanol.html#](https://www.chem.umass.edu/~nermmw/Spectra/irspectra/Methanol/methanol.html#) -->
+
+2. [Problems on Spectroscopy](https://www2.chemistry.msu.edu/faculty/reusch/VirtTxtJml/Spectrpy/InfraRed/problems/irmsprb.htm)
+
+<!-- Not working with https -->
+<!-- 
+3. [http://solidstate.physics.sunysb.edu/book/prob/node111.html](https://solidstate.physics.sunysb.edu/book/prob/node111.html) -->
+
+
+<!-- Domain not found -->
+<!-- 3. [http://www.learnerstv.com/animation/animation.php?ani=56&cat;=Chemistry](http://www.learnerstv.com/animation/animation.php?ani=56&cat;=Chemistry) -->
+
+3. [Vibrational Modes of Small Modecules](https://www.chem.purdue.edu/gchelp/vibs/#)
+
+<!-- Page not found -->
+<!-- 4. [http://orgchem.colorado.edu/hndbksupport/irtutor/tutorial.html](http://orgchem.colorado.edu/hndbksupport/irtutor/tutorial.html) -->
```

Comment:
Broken links have to be updated on [this sheet](https://docs.google.com/spreadsheets/d/1eg8yehiuXJ3ba9EtKIQDgmjJnOPLSilIs5NKvr8nFz0/edit#gid=0) as has been discussed multiple times on the slack channel